types/office-js-preview/index.d.ts

```diff
@@ -11236,7 +11291,7 @@ declare namespace Office {
          * @param callback - Optional. When the method completes, the function passed in the `callback` parameter is called with a single parameter
          *                             of type Office.AsyncResult. The body is provided in the requested format in the `asyncResult.value` property.
          */
-        getAsync(coercionType: Office.CoercionType | string, options?: Office.AsyncContextOptions, callback?: (asyncResult: Office.AsyncResult<string>) => void): void;
```

Comment:
if options is being made mandatory doesn't that need the text update as well above this that describes this parameter? Here and eeverywhere else

Reply:
Kept text and parameter as optional